applications/asset_tracker_v2/src/main.c

```diff
@@ -371,54 +405,30 @@ void on_sub_state_passive(struct app_msg_data *msg)
 		app_cfg = msg->module.data.data.cfg;
 
 		if (app_cfg.active_mode) {
-			LOG_INF("Device mode: Active");
-			LOG_INF("Start data sample timer: %d seconds interval",
-				app_cfg.active_wait_timeout);
-			k_timer_start(&data_sample_timer,
-				      K_SECONDS(app_cfg.active_wait_timeout),
-				      K_SECONDS(app_cfg.active_wait_timeout));
-			k_timer_stop(&movement_timeout_timer);
+			active_mode_timers_start_all();
 			sub_state_set(SUB_STATE_ACTIVE_MODE);
 			return;
 		}
 
-		LOG_INF("Device mode: Passive");
-		LOG_INF("Start movement timeout: %d seconds interval",
-			app_cfg.movement_timeout);
-
-		k_timer_start(&movement_timeout_timer,
-			      K_SECONDS(app_cfg.movement_timeout),
-			      K_SECONDS(app_cfg.movement_timeout));
-		k_timer_stop(&data_sample_timer);
+		passive_mode_timers_start_all();
 	}
 
-	if ((IS_EVENT(msg, sensor, SENSOR_EVT_MOVEMENT_DATA_READY)) ||
-	    (IS_EVENT(msg, ui, UI_EVT_BUTTON_DATA_READY))) {
+	if ((IS_EVENT(msg, ui, UI_EVT_BUTTON_DATA_READY)) ||
```

Comment:
Looks like the number of parentheses could be reduced

Reply:
Unfortunately you get a build warning when omitting a layer of parenthesis here.

Reply:
Interesting, wan't aware